coqbot.toml

```diff
@@ -0,0 +1,22 @@
+[bot]
+name="coqbot"
+email="coqbot@users.noreply.github.com"
+
+[server]
+domain="coqbot.heroku.com"
+
+[github]
+#webhook_secret = "xxx"
+#api_token = "xxx"
+
+[gitlab]
+#api_token = "xxx"
+
+[mappings]
+  [mappings.coq]
+  github = "coq/coq"
+  gitlab = "coq/coq"
+
+  [mappings.math-comp]
+  github = "math-comp/math-comp"
+  gitlab = "math-comp/math-comp"
```

Comment:
We shouldn't hard-code anything about Math-Comp since this is an independent project and they might want to set it up on their own.